hack/verify-typecheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OPENSHIFT_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OPENSHIFT_ROOT}/hack/lib/init.sh"
+
+os::golang::verify_go_version
+
+cd "${OPENSHIFT_ROOT}"
+
+platforms=${PLATFORMS:-'linux/amd64,darwin/amd64,linux/arm,linux/386,linux/arm64,linux/ppc64le,linux/s390x,darwin/386'}
```

Comment:
Add a comment field to the top explaining inputs -- through `$PLATFORMS` as well as command-line args to the script.

Reply:
done